explainaboard/processors/extractive_qa.py

```diff
@@ -10,10 +10,15 @@
 from explainaboard.processors.processor_registry import register_processor
 from explainaboard.tasks import TaskType
 import explainaboard.utils.feature_funcs
+from explainaboard.utils.preprocessor import MLQAPreprocessor
 
 
 @register_processor(TaskType.question_answering_extractive)
 class QAExtractiveProcessor(Processor):
+    def __init__(self):
+        super().__init__()
+        self._preprocessor: MLQAPreprocessor = MLQAPreprocessor()
```

Comment:
Processors should not include data/task-specific information in their member variables (to make sure they remain "stateless"), but the type of preprocessing performed might be task-specific. I think it'd be better for this to be stored somewhere else, maybe having `MetricConfig` included in the `SysInfo` class for example.

Reply:
For future PR